### ISSUE TITLE
The learning rate is too small

### ISSUE THREAD

**Issue by youngfly11** [wkentaro/pytorch-fcn#54]
I found that when you trained the FCN8s model, the learning rate is too small (1e-14). I remember that the learning rate is set to 1e-4 in the original FCN paper. I am a little confused.  
Can you give me some answer? Thank you for advance

**Comment by wkentaro** [comment]
I refer to its journal version:
https://github.com/shelhamer/fcn.berkeleyvision.org/blob/affc85ff245099dbde06609d66ed08d73411f4ad/voc-fcn8s/solver.prototxt#L10

**Comment by youngfly11** [comment]
Thank you for your answer.  The loss does not decay when i train the FCN8s on the CamVid dataset. i am so confused. I have another question.  Do you use the pretrained model or not?

**Comment by wkentaro** [comment]
Yeah, I use pretrained model of fcn16s: https://github.com/wkentaro/pytorch-fcn/blob/master/examples/voc/train_fcn8s.py#L76
which is required to train fcn8s.

**Comment by youngfly11** [comment]
Thanks you so much. Can i directly use the pretrained model of fcn8s?  i remember that you give a url to download the fcn8s_from_caffe.pth in FCN8s.py?

**Comment by wkentaro** [comment]
You can do like below:
```python
import torchfcn
model = torchfcn.FCN8s()
pretrained_model = model.download()
model.load_state_dict(torch.load(pretrained_model))
```
